src/upgradeConfig.js

```diff
@@ -72,19 +74,19 @@ function changePlugins(config) {
 
         if (pluginsToReplace.includes(plugin[0])) {
           if (oldPlugins[plugin[0]]) {
-            plugin[0] = oldPlugins[plugin[0]];
+            plugins[i] = setLooseIfNeeded([oldPlugins[plugin[0]], plugin[1]]);
           } else {
             plugins.splice(i, 1);
             i--;
           }
         }
       } else {
-        if (plugin.indexOf('babel-plugin') !== 0 && plugin[0].indexOf('@babel/') !== 0) {
```

Comment:
`plugin[0].indexOf` looked like a bug -- was preventing `@babel/` prefixes from being recognized correctly

Reply:
submitting this as a separate PR with updated tests in https://github.com/babel/babel-upgrade/pull/48